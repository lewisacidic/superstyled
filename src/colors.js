import style from "./style"

const color = style("color")
const backgroundColor = style("backgroundColor")

export { color, backgroundColor }
