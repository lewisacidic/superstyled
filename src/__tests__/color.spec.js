import React from "react"
import { render } from "./utils"

import { color as c, backgroundColor as bg } from "../colors"

it("should add color", () => {
  const tree = render(<p css={c("red").done()}>test content</p>)
  expect(tree).toHaveStyleRule("color", "red")
  expect(tree).toMatchSnapshot()
})

it("should add background", () => {
  const tree = render(<p css={bg("red").done()}>test content</p>)
  expect(tree).toHaveStyleRule("background-color", "red")
  expect(tree).toMatchSnapshot()
})
