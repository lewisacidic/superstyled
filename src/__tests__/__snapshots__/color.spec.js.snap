// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should add background 1`] = `
.emotion-0 {
  background-color: red;
}

<p
  className="emotion-0"
>
  test content
</p>
`;

exports[`should add color 1`] = `
.emotion-0 {
  color: red;
}

<p
  className="emotion-0"
>
  test content
</p>
`;
